content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,43 @@ environment, then Prometheus is a good choice.
 
-Sensu uses Redis to persist monitoring data, including the Sensu client registry, check results, check execution history, and current event data.
+Sensu stores current and recent event status information and real-time inventory data in an embedded database (etcd) or an external RDBMS (Postgres). All event and time series data is sent to one or more external data platforms for long term storage and analysis (e.g. Prometheus, InfluxDB, TimescaleDB, Elasticsearch, Splunk, Wavefront, etc).
 
 ### Architecture
 
-Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
-RabbitMQ as a transport, Redis for current state, and a separate server for
-processing and API access.
+Sensu is a primarily agent-based monitoring solution ([reference](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/deployment-architecture/)). Sensu agents automatically connect to one or more Sensu backends over an encrypted pub/sub transport (TLS-encrypted websockets), and handle automated registration/de-registration of entities and local service discovery. Sensu [backends](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/) provide the control plane, including the configuration [API](https://docs.sensu.io/sensu-go/latest/api/) and [scheduler](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/). 
 
-All components of a Sensu deployment (RabbitMQ, Redis, and Sensu Server/API) can be clustered for highly available and redundant configurations.
+All components of a Sensu deployment can be clustered for high availability and improved event-processing throughput. Sensu also provides built-in support for federation (including RBAC and other API resource replication) which is recommended for deployments larger than 30,000 nodes (or entities) under management.
 
 ### Summary
-If you have an existing Nagios setup that you wish to scale as-is, or want to take advantage of the automatic registration feature of Sensu, then Sensu is a good choice.
 
-If you want to do whitebox monitoring, or have a very dynamic or cloud based environment, then Prometheus is a good choice.
+Sensu and Prometheus have a few capabilities in common, but they take very different approaches to monitoring. Both offer extensible discovery mechanisms for dynamic cloud-based environments and ephemeral compute platforms (though the underlying mechanisms are quite different). Both provide support for collecting multi-dimensional metrics via labels (and annotations). Both have extensive integrations (and Sensu natively supports collecting metrics from all Prometheus exporters). Where Sensu and Prometheus differ the most is in their use cases.
+
+Where Sensu is better: 
+
+- If you're collecting and processing hybrid observability data (including metrics and events)
+- If you're monitoring hybrid infrastructure (including on-prem servers, public/private cloud, and cloud-native/Kubernetes)
```

Comment:
Different means to a common end I suppose. I don't want to argue this point so I've dropped this from the summary section. Ultimately our goal with this PR is to more accurately tell Sensu's story while being respectful of the Prom project. 

As an aside... my personal thoughts on the matter are that Prometheus is a better experience in a _homogeneous_ K8s environment because the discovery mechanism and scraping "just work" in K8s. On paper you could argue that the same is true with Sensu – our approach to discovery is agent sidecars and/or daemonsets, and data collection "just works" because of the pub/sub transport – but in practice Prometheus is just more optimized for K8s, and visa versa since the K8s platform itself has standardized on Prom for exposing platform metrics. 

Prom does offer SD for basically all-the-platforms out of the box, but the configuration varies from platform-to-platform; Sensu SD works the same on every platform - just run the agent and it auto-registers endpoints. Both have their pros/cons. 

For collecting data, a pull/scrape-based collection workflow isn't quite as seamless outside of K8s as it is on K8s; I suspect many users leverage Prom federation to help here, but opening ports to scrap endpoints across hybrid cloud infrastructure is a complex effort. Conversely, Sensu data collection works the same on every platform thanks to the agent and pub/sub transport, easily traversing complex network topologies including NATs and VPNs. Again you could argue that both have their pros/cons.

TL;DR - if it's true that Prometheus (the most mature CNCF project outside of Kubernetes itself) is optimized for cloud-native/Kubernetes, then by the very nature of optimization it might not necessarily work as seamlessly in other platforms. Conversely, Sensu's more generic approach is not as optimized for Kubernetes, but it works more consistently across all platforms.